bokeh/models/plots.py

```diff
@@ -56,12 +57,8 @@ def _select_helper(args, kwargs):
         selector = kwargs
     return selector
 
-class LayoutBox(Model):
-    ''' Represents an **on-canvas** layout.
-
-    '''
 
-class Plot(Component):
+class Plot(LayoutDOM):
```

Comment:
@mattpap, I know what you mean, but I think it's helpful to distinguish between `LayoutDOM` and `LayoutCanvas`. I think it helps to hint they're not in a class hierarchy (that is `LayoutCanvas` is not a subclass of `Layout`). Am happy for suggestions other than `DOM` as the suffix.